aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -112,14 +81,34 @@ RegistrationHandleRAII Dispatcher::registerDef(FunctionSchema schema) {
 
   OperatorName op_name = schema.operator_name();
 
-  auto op = findOrRegisterSchema_(std::move(schema));
+  const auto found = findSchema(op_name);
+  OperatorHandle op = [&] {
```

Comment:
@bhosmer Renaming here would go against the prevailing naming convention of the entire file, which means I'd have to do a rename of every occurrence of `op` for consistency. Is that really what you want?

Reply:
Ahahaha I'm hearing it like, is that _really_ what you want? 😁 My impression was that `s/handle/op/` was a recent change of yours, but either way, I wouldn't insist. Maybe if I'm in here I'll see what it feels like to make the various indirections more explicit in the names

Reply:
It's not. For example, on master, here's Dispatcher.cpp fragment:

```
std::pair<RegistrationHandleRAII, OperatorHandle> Dispatcher::registerSchema(FunctionSchema schema) {
  // we need a lock to avoid concurrent writes
  std::lock_guard<std::mutex> lock(mutex_);

  OperatorName op_name = schema.operator_name();

  auto op = findOrRegisterSchema_(std::move(schema));

  ++op.operatorIterator_->refcount;
  if (1 == op.operatorIterator_->refcount) {
    // note: call listeners *after* operator is added, i.e. dispatcher is already valid for new op
    listeners_->callOnOperatorRegistered(op);
  }

  return std::make_pair(RegistrationHandleRAII([this, op, op_name] {
    deregisterSchema_(op, op_name);
  }), op);
}
```

op here is an OperatorHandle.